app/main/views/frameworks.py

```diff
@@ -11,3 +12,18 @@ def list_frameworks():
     return jsonify(
         frameworks=[f.serialize() for f in frameworks]
     )
+
+
+@main.route('/frameworks/g-cloud-7/stats', methods=['GET'])
+def get_framework_stats():
+
+    seven_days_ago = datetime.datetime.now() + datetime.timedelta(-7)
```

Comment:
`.utcnow`, it doesn't make much of a difference in this case, but makes it easier to check that we're using UTC everywhere